src/Lang/es/messages.php

```diff
@@ -7,14 +7,13 @@
         'edit'    => 'Editar',
         'yes'     => 'Si',
         'no'      => 'No',
-        'all'      => 'Todo',
```

Comment:
I removed these because they are not used anywhere

Reply:
is used in the sidebar.. to translate "All discussions" and it's separate on purpose to match the needs of the languages, please redo